Cargo.toml

```diff
@@ -43,6 +42,9 @@ chacha20 = "0.9"
 cipher = { version = "0.4", features = ["std"] }
 twofish = "0.7"
 cbc = "0.1"
+totp-lite = "2.0.0"
```

Comment:
@DavidVentura I feel like we should add a `totp` feature, so that those 3 dependencies are optional. That would allow for a more lightweight base library.